### PR TITLE
PBDIT-931: Create linting package for additional linting rules that are now missing with e.g. the base Nest linting package (typescript linting rules)

### PR DESCRIPTION
**Pull request by pieter-pon**
https://pondevelopment.atlassian.net/browse/PBDIT-931

Usefull for using as extra lint package beside other linting packages as standardjs (like microservices based on expressjs and framework agnostic packages) or typescript lint rules (like NestJS services)

### CODE REVIEW

**Review by BartPon — APPROVED** [review]

File: .nvmrc
Comment (stephantromer-pon) [inline .nvmrc:1]: Do we need this here?

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+            }
+        ],
+        'no-useless-concat': 'error',
+        'no-nested-ternary': 'error',
+        'max-len': [
+            'error',
+            {
+                'code': 120,
+                'comments': 120,
+                'tabWidth': 4,
```
Comment (stephantromer-pon) [inline .eslintrc.js]: We're using tab width 2 right?

Comment (pieter-pon) [reply .eslintrc.js]: i will remove tabWidth, it's something this lint package shouldnt check, these formatting rules are for other linting packages.

File: .nvmrc
Comment (pieter-pon) [reply .nvmrc:1]: It's good to always add it, so if you install dependencies, run linting, etc. you use a preferred node version

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        'plugin:n/recommended',
+        'plugin:jsdoc/recommended',
+        'plugin:sonarjs/recommended'
+    ],
+    'parserOptions': {
+        'sourceType': 'module',
+        'ecmaVersion': 'latest'
+    },
+    'globals': {
+        'fetch': false,
```
Comment (pieter-pon) [inline .eslintrc.js:14]: Check of this list is still needed, should be native available

**Pull request merged**